app/presenters/work_combined_audio_derivatives.rb

```diff
@@ -45,4 +45,22 @@ def derivatives_up_to_date?
     CombinedAudioDerivativeCreator.new(model).fingerprint == combined_audio_fingerprint
   end
 
+
+  def combined_audio_creation_status_and_date
+    @combined_audio_creation_status ||= begin
+      status = combined_audio_creation_status
+      return nil if status.nil?
+      "#{status} as of #{combined_audio_creation_date}"
+    end
+  end
+
+  def combined_audio_creation_date
+    model&.oral_history_content&.combined_audio_derivatives_creation_status_changed_at
+  end
```

Comment:
this is not necessarily a "creation" date, right? It could be the date it was set to "in progress" or the date it was marked as an error...

Reply:
Good point!